pkg/registry/input_stream.go

```diff
@@ -142,7 +142,7 @@ func (r *ReplacesInputStream) Next() (*ImageInput, error) {
 				delete(r.packages, pkg)
 			}
 
-			return image, nil
+			return image, utilerrors.NewAggregate(errs)
```

Comment:
this fix looks incorrect
the way this should work is, as long as we find a bundle that works for at least one package, don't return an error 
eventually next will have no bundles, or bundles that cause errors on add

Reply:
I've opened #523, which I believe fixes the regression. It also adds a test that fails before/succeeds after the change.

@anik120 PTAL, and let me know if you agree with the fix.